packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -250,6 +256,7 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
     const rowFields = (
       <RowFields
         rowClassNames={this._rowClassNames}
+        rowHeaderId={`${this._rowHeaderId}`}
```

Comment:
nit, these should already be strings and therefore don't need to be wrapper in backticks (same below)